packages/mobile/src/components/user-badges/UserBadgesV2.tsx

```diff
@@ -22,10 +22,23 @@ export const UserBadgesV2 = (props: UserBadgesProps) => {
 
   const { tier } = useTierAndVerifiedForUser(userId)
 
+  const bonkToken = getTokenBySymbol(env, 'BONK')
```

Comment:
assuming we should feature flag?

Reply:
do we need to if nobody will have bonk on prod? we don't even have bonk user banks yet.

Reply:
yea let's feature flag under some v1 flag and i'll use the same flag in my next PR